drivers/can/can_mcux_flexcan.c

```diff
@@ -164,14 +167,14 @@ static int mcux_flexcan_set_mode(const struct device *dev, enum can_mode mode)
 	FLEXCAN_GetDefaultConfig(&flexcan_config);
 	flexcan_config.clkSrc = config->clk_source;
 	flexcan_config.baudRate = clock_freq /
-	      (1U + data->timing.propSeg + data->timing.phaseSeg1 +
-	       data->timing.phaseSeg2) / data->timing.preDivider;
+	      (1U + data->timing.prop_seg + data->timing.phase_seg1 +
+	       data->timing.phase_seg2) / data->timing.prescaler;
 	flexcan_config.enableIndividMask = true;
 
-	flexcan_config.timingConfig.rJumpwidth = data->timing.rJumpwidth;
-	flexcan_config.timingConfig.propSeg = data->timing.propSeg;
-	flexcan_config.timingConfig.phaseSeg1 = data->timing.phaseSeg1;
-	flexcan_config.timingConfig.phaseSeg2 = data->timing.phaseSeg2;
+	flexcan_config.timingConfig.rJumpwidth = data->timing.sjw - 1U;
```

Comment:
This wasn't here before either, but does this code segment not need to set `flexcan_config.timingConfig.preDivider` as well?

Reply:
`FLEXCAN_Init()` call `FLEXCAN_SetBaudRate()`, wich calculate `timingConfig.preDivider` (`modules\hal\nxp\mcux\drivers\imx\fsl_flexcan.c:683`).

Reply:
Aight, thanks for clearing it up